dali/test/python/test_operator_rotate.py

```diff
@@ -180,3 +223,145 @@ def test_gpu_vs_cpu():
   for test in run_cases("gpu", "cpu", 1):
     yield test
 
+
+def maximum_array(arrays):
+  assert(len(arrays))
+  acc_max = arrays[0]
+  for array in arrays[1:]:
+    acc_max = np.maximum(acc_max, array)
+  return acc_max
```

Comment:
```suggestion
  return np.max(arrays, axis=0)
```
It will automatically treat the outer list as an extra dimension.

Reply:
done